### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request provides groups that allow given users to manage the assessment images buckets (defined in [cisagov/cool-images-assessment-images]) in the environments specified.

## 💭 Motivation and context ##

Currently [cisagov/cool-images-assessment-images] is configured with a service user to upload images to the buckets in the configuration. We are moving away from that configuration and so we need to give individual IAM users equivalent capabilities. This configuration adds users to groups created to allow assumption of the `AssessmentImagesBucketFullAccess` roles defined in [cisagov/cool-images-assessment-images].

## 🧪 Testing ##

Automated tests pass. We were able to verify that when this configuration was applied the users were able to access the given S3 buckets in the expected manner.

## ℹ To-do ##

- [x] Add an issue to decouple the two environments (production and staging) in this configuration. This is reliant on https://github.com/cisagov/cool-images-assessment-images/issues/13 and I will create that issue and document it once this pull request is closer to ready-to-merge.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

[cisagov/cool-images-assessment-images]: https://github.com/cisagov/cool-images-assessment-images

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
Very nice!

File: README.md [hunk 1]
```diff
@@ -51,41 +56,39 @@ No modules.
...
-| aws\_availability\_zone | The AWS availability zone to deploy into (e.g. a, b, c, etc.). | `string` | `"a"` | no |
+| assessment\_images\_managers\_group\_name | The base name of the group to be created for assessment images manager users in each Images account. This value has the environment name appended to it for each environment. | `string` | `"assessment_images_managers"` | no |
+| assume\_images\_assessmentimagesbucketfullaccess\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows full access to the assessment images bucket in an Images account. | `string` | `"The IAM policy that allows assumption of the role that allows full access to the assessment images bucket in an Images account."` | no |
+| assume\_images\_assessmentimagesbucketfullaccess\_policy\_name | The base name to assign the IAM policies that allows assumption of the role that allows full access to the assessment images bucket in an Images account This value has the environment name appended to it for each environment. | `string` | `"Images-AssumeAssessmentImagesBucketFullAccess"` | no |
```
Comment (dav3r) [inline README.md]: Grammar nitpicks:
```suggestion
| assume\_images\_assessmentimagesbucketfullaccess\_policy\_name | The base name to assign the IAM policies that allow assumption of the role that allows full access to the assessment images bucket in an Images account. This value has the environment name appended to it for each environment. | `string` | `"Images-AssumeAssessmentImagesBucketFullAccess"` | no |
```

File: variables.tf [hunk 2]
```diff
@@ -4,30 +4,43 @@
...
   type        = string
-  description = "The AWS availability zone to deploy into (e.g. a, b, c, etc.)."
-  default     = "a"
+  description = "The description to associate with the IAM policy that allows assumption of the role that allows full access to the assessment images bucket in an Images account."
+  default     = "The IAM policy that allows assumption of the role that allows full access to the assessment images bucket in an Images account."
+}
+
+variable "assume_images_assessmentimagesbucketfullaccess_policy_name" {
+  type        = string
+  description = "The base name to assign the IAM policies that allows assumption of the role that allows full access to the assessment images bucket in an Images account This value has the environment name appended to it for each environment."
```
Comment (dav3r) [inline variables.tf]:
```suggestion
  description = "The base name to assign the IAM policies that allow assumption of the role that allows full access to the assessment images bucket in an Images account. This value has the environment name appended to it for each environment."
```

**Review by dav3r — APPROVED** [review]
Great job with this!  💪 💼 

I only found the tiniest of nits to pick.

**Pull request merged**